install/startup.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+echo "Wait for Kubernetes context to be set and ready"
+
+max_retry_count=5
```

Comment:
Should we make this configurable via ENV ? so we can control the delay without creating a new image